lekko_client/clients/cached_git_client.py

```diff
@@ -107,3 +109,11 @@ def close(self) -> None:
         if self.watcher:
             self.watcher.stop()  # type: ignore
             self.watcher.join()
+
+
+def _get_blob_sha(path: str) -> bytes:  # pragma: no cover
+    """Finds or computes the blob sha of a file"""
```

Comment:
supernit (and I have never seen it being followed 100%), but docstring guideline says that docstring should be "a command, not a description", so "find" instead of "finds" :) See https://peps.python.org/pep-0257/#one-line-docstrings